spm_tutorials/spm-FaceRep_smdl.json

```diff
@@ -0,0 +1,101 @@
+{
+    "BIDSModelVersion": "1.0.0",
+    "Name": "SPM_FaceRepetition",
+    "Description": "Look at a) main effect of seeing task, b) see if repetition affects BOLD response, c) effect of motion",
+    "Input": {
+        "Include": {
+            "task": "FaceRepetition"
+        }
+    },
+    "Nodes": [
+        {
+            "Level": "subject",
+            "Model": {
+                "X": [
+                    "trial_type.nonfamous_1",
+                    "trial_type.nonfamous_2",
+                    "trial_type.famous_1",
+                    "trial_type.famous_2",
+                    "trans_x",
+                    "trans_y",
+                    "trans_z",
+                    "rot_x",
+                    "rot_y",
+                    "rot_z"
+                ],
+                "ModelType": "glm",
+                "Options": {
+                    "high_pass_filter_cutoff_secs": 128
+                },
+                "Software": {
+                    "SPM": {
+                        "HRF_type": "informed_basis_set",
+                        "whitening": "AR(1)"
+                    }
+                }
+            },
+            "Contrasts": [
+                {
+                    "Name": "Positive effect of condition_1",
+                    "ConditionList": [
+                        "nonfamous_1_bf1",
+                        "nonfamous_2_bf1",
+                        "famous_1_bf1",
+                        "famous_2_bf1"
+                    ],
+                    "Weights": [
+                        1,
+                        1,
+                        1,
+                        1
+                    ],
+                    "Type": "t"
+                },
+                {
+                    "Name": "Main effect of Repetition",
+                    "ConditionList": [
+                        "nonfamous_1_bf1",
```

Comment:
For cases when several regressors coming from the same predictors are passed the the next level, I have adopted this notatation to denote each "basis" function.

Given how rarely temporal and dispersion derivatives (or FIR models) are used in practice I now feel that these examples eventhough they come from tutorials, might actually represent the 20% from BIDS "pareto principle" so I wonder how much sense it makes to keep them as they might be more cofusing than anything else.

Reply:
Perhaps it's worthwhile thinking if they are used less in practice because most people avoid the complications of using them rather than they are not useful. We have used FIR models quite often actually across a host of experiments.

Reply:
oh I hear you!! :-)

Actually I also meant to say but forgot to type; that even if they are not common, it would be very unwise to write specs in a way that completley ignores those use cases and potentially does not make it possible to use FIR or "less conventional" options.